### PR TITLE
mobile menu

### PR DESCRIPTION
**Pull request by borbesaur**

### CODE REVIEW

File: app/components/views/Home/index.tsx [hunk 1]
```diff
@@ -121,17 +122,37 @@ const useProvider = (): [
   ];
 };
 
+const useIsMobile = (): { isMobile: boolean } => {
```
Comment (Atmosfearful) [inline app/components/views/Home/index.tsx:125]: Can we just use CSS breakpoints to show/hide the `.mobileMenu` class?

File: app/components/views/Home/MobileMenu/index.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import { FC, useState } from "react";
+import { Link } from "react-router-dom";
+import { IconButton, Menu, MenuItem, Zoom } from "@mui/material";
```
Comment (Atmosfearful) [inline app/components/views/Home/MobileMenu/index.tsx:3]: Please use this import syntax, which iirc has better tree shaking.

```ts
import IconButton from '@mui/material/IconButton';
```

**Review by Atmosfearful — CHANGES_REQUESTED** [review]
Awesome work!
Just a couple changes.

- [ ] To align with the typography system, can we use `typography.button` styles on the mobile menu item buttons?
- [ ] I think its worth tweaking the icon so it appears (visually) aligned with the card horizontally, and centered on the logo vertically

Untitled

File: app/components/views/Home/index.tsx (hunk 1 above)
Comment (borbesaur) [reply app/components/views/Home/index.tsx]: no prob

File: app/components/views/Home/MobileMenu/index.tsx (hunk 2 above)
Comment (borbesaur) [reply app/components/views/Home/MobileMenu/index.tsx]: done

**Review by Atmosfearful — APPROVED** [review]

**Pull request merged**